.github/workflows/build-world.yaml

```diff
@@ -24,6 +24,8 @@ jobs:
       options: |
         --cap-add NET_ADMIN --cap-add SYS_ADMIN --security-opt seccomp=unconfined --security-opt apparmor:unconfined
 
+    timeout-minutes: 2880 # 2 days
```

Comment:
purposely over estimated until we see green

Reply:
I mean, eventually it will take 2 days to build Wolfi from scratch anyway.  The more packages we add, the longer it will take ;)